lib/widgets/quantity_selector.dart

```diff
@@ -43,6 +45,7 @@ class _QuantitySelectorState extends State<QuantitySelector> {
 
   @override
   Widget build(BuildContext context) {
```

Comment:
I don’t see any need to modify this function. Can you undo the edits to this function?